components/molecules/InterstitialScreen/InterstitialScreen.scss

```
@@ -1,14 +1,59 @@
 .uic--interstitial-screen {
   background-color: $royal;
 
+  &__action-bar {
+    padding: 1.3rem 1.5rem;
```

```suggestion
    padding: 1.3rem 2rem;
```
Pretty sure it should be 2rem away from the side. It should line up with the edge of like where the log out button or like the analyze plan buttons are